### PR TITLE
fix(ADA-1359): Make scrollbar visible all the time

### PR DESCRIPTION
**Pull request by valentinStoicaK**
Root cause:
- Scrollbar was not visible;

PR description:
- I've modified the theme.css so that the scrollbar was similar with the one in the transcript plugin;
- I've removed scroll handling from the playlist plugin;
- I've removed a comma from playlist-wrapper.css, because of the errors that were created by it;

PS: I don't have the possibility of testing it on Safari and QA informed me that the scrollbar was not visible at all on safari before the fix;

### CODE REVIEW

File: src/components/playlist-wrapper/index.tsx [hunk 1]
```diff
@@ -90,19 +77,11 @@ export const PlaylistWrapper = withText(translates)(
...
-      setScrolling(false);
-    }, []);
-    
-    const handleKeyDown = useCallback((event: KeyboardEvent) => {
-      if (event.keyCode === KeyMap.UP || event.keyCode === KeyMap.DOWN || event.keyCode === KeyMap.LEFT || event.keyCode === KeyMap.RIGHT) {
-        setScrolling(true);
-      }
-    }, []);
+
+    const handleFocus = useCallback(() => {}, []);
```
Comment (semarche-kaltura) [inline src/components/playlist-wrapper/index.tsx:81]: if that's only `noop` methods, why keep them? Can we remove `handleFocus`, `handleBlur` and `handleKeyDown` methods?

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/components/playlist-wrapper/index.tsx (hunk 1 above)
Comment (valentinStoicaK) [reply src/components/playlist-wrapper/index.tsx]: Done

**Comment by SivanA-Kaltura** [comment]
@valentinStoicaK please change the version of cypress in package.json to 13.13.1, run yarn install, and commit package.json and yarn.lock. this should solve the firefox timeout that's failing the tests.

**Comment by valentinStoicaK** [comment]
@SivanA-Kaltura Done!

**Review by semarche-kaltura — APPROVED** [review]

**Pull request merged**